vue/src/ManageInstallTagCode/ManageInstallTagCode.vue

```diff
@@ -53,7 +53,7 @@
       <br />
       <a
         target="_blank"
-        href="https://matomo.org/faq/tag-manager/data-layer-in-matomo-tag-manager/"
+        href="{{ externalRawLink('https://matomo.org/faq/tag-manager/data-layer-in-matomo-tag-manager/') }}"
```

Comment:
@bx80 you need to change to `:href` and related changes to make it work

Reply:
Done :+1: